tests/ts/chfn/gecos

```diff
@@ -22,6 +22,8 @@ TS_DESC="gecos"
 . "$TS_TOPDIR"/functions.sh
 ts_init "$*"
 
+test -e /etc/libuser.conf || export LIBUSER_CONF=/dev/null
```

Comment:
What unconditionally setting the variable?
Then we always have a well-known and working config

Reply:
Well, I don't mind setting it to `/dev/null` unconditionally, but maybe @karelzak have any thoughts on it?!

Reply:
Setting it unconditionally is probably no problem.

Reply:
thanks